astroarch_build.sh

```diff
@@ -44,7 +44,8 @@ pacman -Syu base-devel pipewire-jack gnu-free-fonts pipewire-media-session \
 	ksystemlog discover kwalletmanager kgpg qt5-serialbus \
 	qt5-serialport qt5ct udisks2-qt5 xorg-fonts-misc fuse2 \
 	fortune-mod cowsay pacman-contrib arandr neofetch \
-	astromonitor kscreen sddm-kcm --noconfirm --ask 4
+	astromonitor kscreen sddm-kcm vsftpd plymouth \
+	bluez bluez-utils bluez-hid2hci bluedevil --noconfirm --ask 4
```

Comment:
bluetooth should not be part of the base image IMO. Just added as documentation in case somebody wants to use it